elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -473,8 +480,15 @@ export class PeerPool extends EventEmitter {
 		return this.getAllPeers().map(peer => peer.peerInfo);
 	}
 
-	public getAllPeers(): ReadonlyArray<Peer> {
-		return [...this._peerMap.values()];
+	public getAllPeers(
+		kind?: typeof OutboundPeer | typeof InboundPeer,
```

Comment:
maybe can use `unknown` here if the idea is to allow other types of Peers in the future.